<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerGui</class>
 <widget class="QWidget" name="VideoPlayerGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tchoupi</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QMenuBar" name="menuBar">
      <widget class="QMenu" name="menusrt">
       <property name="title">
        <string>Srt</string>
       </property>
       <addaction name="fromOpensubtitlesAct"/>
      </widget>
      <addaction name="menusrt"/>
     </widget>
    </item>
    <item>
     <widget class="QVideoWidget" name="videoWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="50,0,50">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="labelLayout">
        <property name="spacing">
         <number>2</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Slider" name="slider">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="errorLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="fromOpensubtitlesAct">
   <property name="text">
    <string>from openSubtitles...</string>
   </property>
   <property name="toolTip">
    <string>download the srt file from openSubtitles</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
